frontend/app/views/fragments/event/stats.scala.html

```diff
@@ -0,0 +1,28 @@
+@(event: model.Eventbrite.RichEvent)
+
+@import views.support.Dates._
+
+<div class="stats-listing">
+    <div class="stat-item">
+        <div class="stat-item__first">
+            <i class="icon icon-early_booking"></i>
+        </div>
+        <div class="stat-item__second">
+            <span itemprop="startDate" content="@event.start">@prettyDateWithTimeAndDayName(event.start)</span>
+        </div>
+    </div>
+    <div class="stat-item">
+        <div class="stat-item__first">
+            <i class="icon icon-location"></i>
+        </div>
+        <div class="stat-item__second copy" itemprop="location" itemscope itemtype="http://schema.org/Place">
+            @if(event.venue.name) {@event.venue.name}@if(event.venue.address){, }
```

Comment:
I think this has been copied in - this has always concerned me how ugly this is with the whitespace. Maybe needs a future think?